### PR TITLE
[packaging] Clarify feature disparity

### PR DESCRIPTION
**Pull request by Ferenc-**
Intended for fixing https://github.com/flathub/app.organicmaps.desktop/issues/26

Signed-off-by: Ferenc Géczi <ferenc.gm@gmail.com>

### CODE REVIEW

File: packaging/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -28,6 +28,9 @@
     </p>
     <p>## Features</p>
     <p>
+      The Linux version of Organic Maps has not reached feature parity
```
Comment (endim8) [inline packaging/app.organicmaps.desktop.metainfo.xml:31]: Probably best to call it the 'Desktop version' as it's also usable on MacOS

**Review by endim8 — CHANGES_REQUESTED** [review]
Small nit but otherwise LGTM! :)

File: packaging/app.organicmaps.desktop.metainfo.xml (hunk 1 above)
Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml:31]: > Probably best to call it the 'Desktop version' as it's also usable on MacOS

To the best of my knowledge appstream metadata is not usable on MacOS, and I believe only Linux users will read this. And my understanding is that creating a convergent UI which is not for desktop only is [at least still on the talbe](https://github.com/organicmaps/organicmaps/discussions/4408). And this metainfo would be suitable for that hypothetical scenario as well.

Comment (endim8) [reply packaging/app.organicmaps.desktop.metainfo.xml:31]: alright fair enough :)

**Review by endim8 — APPROVED** [review]

**Review by biodranik — APPROVED** [review]

**Pull request merged**